ExponentiationBySquaring.java

```diff
@@ -0,0 +1,31 @@
+public class ExponentiationBySquaring {
```

Comment:
Hi @rishier827 
There are two java files in the commit,

Reply:
okay i will delete 1 file